code/datums/components/armor_plate.dm

```diff
@@ -1,91 +1,105 @@
 /datum/component/armor_plate
+	/// Current amount of upgrades
 	var/amount = 0
+	/// Maximum amount of upgrades
 	var/maxamount = 3
```

Comment:
You have mixed use of `maxamount` and `max_amount`. `max_amount` would be preferred variable-name wise.

Reply:
I done goofed, pushing fixes.